pkg/stagemanager/stage_manager.go

```diff
@@ -392,6 +392,8 @@ func (stm *StageManager) Stop() {
 	// since the new started server may have the same pid file
 	if preState != Upgrading {
 		pid.RemovePidFile()
+		// stop reconfigure domain socket
+		syscall.Unlink(types.ReconfigureDomainSocket)
```

Comment:
Use `server.StopReconfigureHandler()` and remove the judgement condition  about `state`.

Reply:
cycle import